content/https/https-3/contents.lr

```diff
@@ -0,0 +1,15 @@
+_model: question
+---
+title: What's the difference between Tor and DoH (DNS over HTTPS)?
+---
+seo_slug: difference-between-tor-doh
+---
+description:
+DNS over HTTPS (DoH) is a protocol for performing remote DNS resolution via the HTTPS protocol. A goal of the method is to increase user privacy and security by preventing eavesdropping and manipulation of DNS data by man-in-the-middle attacks by using the HTTPS protocol to encrypt the data between the DoH client and the DoH-base DNS resolver. Encryption by itself does not protect privacy, encryption is simply a method to obfuscate the data.
+DoH specifies how to send and receive DNS queries over HTTPS. Server configuration is performed out of band, and the connection with the resolver is secured as any other HTTPS traffic. DoH is mostly targeted at web browsers and does not have the potential for improving the privacy properties of transactions between recursive resolvers and authoritative nameservers.
+One possible benefit from a user standpoint is that since DNS queries can be intermingled with normal web traffic, DoH might prove more difficult to block.
+
+Tor is a network of virtual tunnels that allows you to improve your privacy and security on the Internet. Tor works by sending your traffic through three random servers (also known as relays) in the Tor network. The last relay in the circuit (the “exit relay”) then sends the traffic out onto the public Internet.

```

Comment:
Maybe it would be good to add more links to other documents, I see this part repeats a bit content that we surely have somewhere else, we should try to reduce duplicated content

Reply:
Just improved the answer, please check it.
And may I add exactly this DoH explanation to the Glossary page?